tasks/connectors/ms_defender_tvm/ms_defender_tvm.rb

```diff
@@ -121,111 +120,120 @@ def run(opts)
           vuln_json = vuln_json_response["value"]
 
           vuln_json.each do |vuln|
-            vuln_cve = vuln.fetch("cveId")
-            scanner_id = vuln_cve
-            if vuln_cve.start_with?("CVE")
-              vuln_cve = vuln_cve.strip
-            else
-              vuln_name = vuln_cve
-              vuln_cve = nil
-            end
-
-            machine_id = vuln.fetch("deviceId")
-            fqdn = vuln.fetch("deviceName")
-
-            # Get the asset details & craft them into a hash
-            asset = {
-              "external_id" => machine_id,
-              "fqdn" => fqdn,
-              "hostname" => fqdn.split(".")[0],
-              "os" => vuln.fetch("osPlatform"),
-              "os_version" => vuln.fetch("osVersion"),
-              "first_seen" => vuln.fetch("firstSeenTimestamp"),
-              "last_seen" => vuln.fetch("lastSeenTimestamp")
-            }
-
-            # Construct tags
-            tags = []
-            tags << "MSDefenderTvm"
-            tags << "rbacGroup: #{vuln.fetch('rbacGroupName')}" unless vuln.fetch("rbacGroupName").nil?
-
-            # Add them to our asset hash
-            asset["tags"] = tags
-            vuln_score = (vuln_severity[vuln.fetch("vulnerabilitySeverityLevel")] || 0).to_i
-
-            if asset_id.nil?
-              print_debug "setting machine id for first asset"
-              asset_id = machine_id
-              asset_count += 1
-              print_debug "asset count = #{asset_count}"
-            end
-
-            if asset_id.!= machine_id
-              if asset_count == batch_page_size
-                submit_count += 1
-                print_debug "#{submit_count} about to upload file"
-                filename = "microsoft_tvm_kdi_#{submit_count}.json"
-                kdi_upload("#{$basedir}/#{output_directory}", filename, @kenna_connector_id, @kenna_api_host, @kenna_api_key, false, 3)
-                asset_count = 0
+              vuln_cve = vuln.fetch("cveId")
```

Comment:
There are too many spaces (4). Rubocop is configured so that tab should be 2 spaces. When you fix it, the Lint Code Base check will pass

Reply:
done